recipe_scrapers/jocooks.py

```diff
@@ -0,0 +1,54 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class JoCooks(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "jocooks.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".wprm-recipe-ingredient-group h4",
+            ".wprm-recipe-ingredient",
+        )
+
+    def instructions(self):
+        return self.schema.instructions()
+
+    def ratings(self):
+        return self.schema.ratings()
+
+    def cuisine(self):
+        return self.schema.cuisine()
+
+    def description(self):
+        return self.schema.description()
+
+    def nutrients(self):
+        return self.schema.nutrients()
```

Comment:
```suggestion
    def nutrients(self):
        return self.schema.nutrients()

    def equipment(self):
        equipment_list = self.soup.find("ul", {"class": "wprm-recipe-equipment-list"})
        if not equipment_list:
            return None

        equipment = []
        for item in equipment_list.find_all("li", {"class": "wprm-recipe-equipment-item"}):
            equipment.append(item.text)

        return equipment
```

...and an implementation for `equipment`.